src/Couchbase.Lite.Tests.Shared/QueryTest.cs

```diff
@@ -58,6 +60,74 @@ public QueryTest(ITestOutputHelper output) : base(output)
         }
 #endif
 
+        [Fact]
+        public void TestQueryDocumentExpiration()
+        {
+            var dto20 = DateTimeOffset.Now.AddSeconds(20);
+            var dto30 = DateTimeOffset.Now.AddSeconds(30);
+            var dto40 = DateTimeOffset.Now.AddSeconds(40);
+            var dto60InMS = DateTimeOffset.Now.AddSeconds(60).ToUnixTimeMilliseconds();

```

Comment:
Why is this in milliseconds, when the value is stored in the database in seconds?  Please confirm which is correct and make them match.  If the value is stored in seconds, then this test will pass falsely, since milliseconds will always be much higher.

Reply:
Jens told me currently the expiration API uses seconds and N1QL is in ms. And possible he will change the expiration API to use ms for consistency...